src/pages/generate/pages/loading/LoadingPage.css.ts

```
@@ -55,6 +55,14 @@ export const loadText = style({
   ...fontStyle('caption_r_12'),
   color: colorVars.color.gray500,
   textAlign: 'center',
+
+  '@media': {
```

ESLint

[error] Replace `'@media'` with `"@media"`

(prettier/prettier)